src/gui/statusbar.cpp

```diff
@@ -158,12 +158,14 @@ void StatusBar::showRestartRequired()
 {
     // Restart required notification
     const QString restartText = tr("qBittorrent needs to be restarted!");
-    QLabel *restartIconLbl = new QLabel(this);
-    restartIconLbl->setPixmap(style()->standardPixmap(QStyle::SP_MessageBoxWarning));
+
+    const QPixmap pixmap = style()->standardIcon(QStyle::SP_MessageBoxWarning).pixmap(Utils::Gui::smallIconSize());
```

Comment:
I didn't delve into it much. I hope there are no problems related to this or that pixmap size.